internal/pkg/action/operator_uninstall.go

```diff
@@ -81,7 +114,7 @@ func (u *OperatorUninstall) Run(ctx context.Context) error {
 	// OLM puts an ownerref on every namespaced resource to the CSV,
 	// and an owner label on every cluster scoped resource. When CSV is deleted
 	// kube and olm gc will remove all the referenced resources.
-	if err := u.deleteObjects(ctx, csv); err != nil {
+	if err := u.deleteObjects(ctx, csvObj); err != nil {
```

Comment:
If we come out of the CSV fetching with a `nil` csvObj, that means we will have a `nil` CSV and no CRDs to delete (since we find the CRDs from the CSV), right?

Do we need to check to see if the CSV is nil before trying to delete it here?

If it is `nil`, I think that means we'll only be deleting the subscription and possibly the operator group and NOT erroring out on a missing CSV? Seems reasonable since its possible there's a problem with the subscription that prevented the CSV from being created.

Reply:
A nil check should be performed in `deleteObjects()` like in https://github.com/operator-framework/operator-sdk/pull/4303.